ls1intum/Artemis
# `Development`: Skip build triggers for programming exercises during file imports

### Checklist
#### General


#### Server

#### Changes affecting Programming Exercises

### Motivation and Context


This PR fixes an issue introduced by #7329 that broke two server tests. The assumption in that PR was flawed and is now fixed, we only skip the triggers for the builds if we import an exercise from a file.

### Description

Changed the name of the argument to make it clearer on how to use it and fixed it's usage.
### Steps for Testing

Prerequisites:
- 1 Instructor
- 2 Students
- 1 Programming Exercise with Complaints enabled

1. Run the server tests and check if all pass
2. For functionality, refer to the testing steps from #7329 

### Review Progress






#### Performance Review
#### Code Review
#### Manual Tests
#### Exam Mode Test

### Test Coverage







### Screenshots
